README.md

```diff
@@ -5,46 +5,196 @@
 It supports pre-trained models from the [Open Model Zoo], along with 100+ open
 source and public models in popular formats such as TensorFlow, ONNX, PaddlePaddle, MXNet, Caffe, Kaldi.
 
-## Repository components
-* [OpenVINO™ Runtime]
-* [Model Optimizer]
-* [Post-Training Optimization Tool]
-* [Samples]
+### Components
+* [OpenVINO™ Runtime] - is a set of C++ libraries with C and Python bindings providing a common API to deliver inference solutions on the platform of your choice.
+    * [core](https://github.com/openvinotoolkit/openvino/tree/update_readme/src/core) - provides the base API for model representation and modification.
+    * [inference](https://github.com/openvinotoolkit/openvino/tree/update_readme/src/inference) - provides an API to infer models on device.
+    * [transformations](https://github.com/openvinotoolkit/openvino/tree/master/src/common/transformations) - contains the set of common transformations which are used in OpenVINO plugins.
+    * [low precision transformations](https://github.com/openvinotoolkit/openvino/tree/master/src/common/low_precision_transformations) - contains the set of transformations which are used in low precision models
+    * [bindings](https://github.com/openvinotoolkit/openvino/tree/master/src/bindings) - contains all awailable OpenVINO bindings which are maintained by OpenVINO team.
+        * [c](https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/c) - provides C API for OpenVINO™ Runtime
+        * [python](https://github.com/openvinotoolkit/openvino/tree/master/src/bindings/python) - Python API for OpenVINO™ Runtime
+* [Plugins](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins) - contains OpenVINO plugins which are maintained in open-source by OpenVINO team. For more information please taje a look to the [list of supported devices](#supported-hardware-matrix).
+* [Frontends](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends) - contains available OpenVINO frontends which allow to read model from native framework format.
+    * [IR](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/ir) allows to read model from OpenVINO IR format.
+    * [ONNX](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx) allows to read native ONNX models.
+    * [Paddle Paddle](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/paddle) allows to read native Paddle Paddle models.
+    * [Tensorflow](https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/tensorflow) allows to read native Tensorflow models.
+* [Model Optimizer] - is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
```

Comment:
This is a nice example how the others should be like. As a new-comer, I see a brief description regarding the model optimizer and lots of examples. And, everything is in one place, without clicking so many times! Good one!